L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py

```diff
@@ -18,5 +18,5 @@
     unpackEcalMask = cms.bool(False),
     unpackHcalMask = cms.bool(False),
     # See UCTLayer1.hh for firmware version
-    firmwareVersion = cms.int32(1),
+    firmwareVersion = cms.int32(3),
```

Comment:
it is part of the stage2 L1 trigger emulation, not sure what you are asking specifically